renovate.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    ":automergeLinters",
+    ":automergeTypes",
```

Comment:
Automerge sounds like something we should never ever allow or do.

Reply:
1. This is very conservative and only permits it for ESLint, Prettier, and `@types/*` packages.  If the CI passes, there's no reason not to merge.  That's what any human would do.
2. It won't actually allow it anyway because the PR still requires approval per the branch protection rule.  Without changes on GitHub, all it will do is effectively check the box on the PR.

Renovate has a decent [discussion of the automerge feature on their website](https://docs.renovatebot.com/key-concepts/automerge/).

Reply:
This was a serious review comment which you just decline, added yourself, self approved an now merged.


Please revert this asap.

Reply:
> That's what any human would do.

I hope for sure, that is not what any review does. At least the upstream changes should be reviewed. Something renovate will no do for you, as it is not an human.

../Frenck

Reply:
> I hope for sure, that is not what any reviewer (or you) does. At least the upstream changes should be reviewed. Something renovate will not do for you, as it is not an human.

I did not intend to imply that the release notes should not be reviewed.  I certainly do that every time.  I simply meant that in those particular cases, the CI lint is all that is needed in terms of testing.

Again, I included those rules in a way that was simply meant as an extra click time saver on a small subset of very low risk packages.  No actual automatic merging was configured.

In any case, I removed all mentions of automerge in #15743.